src/assets/i18n/en.json5

```diff
@@ -2124,6 +2126,36 @@
 
   "home.top-level-communities.help": "Select a community to browse its collections.",
 
+  "info.accessibility-settings.breadcrumbs": "Accessibility settings",
+
+  "info.accessibility-settings.disableNotificationTimeOut.label": "Hide notifications automatically",
+
+  "info.accessibility-settings.disableNotificationTimeOut.hint": "When this toggle is activated, notifications will remain until manually closed.",
+
+  "info.accessibility-settings.liveRegionTimeOut.label": "Live region time-out",
```

Comment:
I think we should call this "ARIA live region time out (in seconds)".  These live regions are ARIA live regions: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions